config/k8s/worker-deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
```

Comment:
Let's add the license header and also an description including that we count on these to restart themeselves.